### ISSUE TITLE
3rd Consecutive Bonus (3CB)

### ISSUE THREAD

**Issue by josiah-leinbach** [gryphonshafer/Quizzing-Rule-Book#199]
Similar to a 3rd Person Bonus (3PB), a 3rd Consecutive Bonus (3CB) would give an additional 10 points to a team that answers three consecutive questions correctly. Those three could come from a single quizzer or three different quizzers. This scoring addition would have several competitive advantages to quizzing:

1. Currently, there are only two ways to achieve additional points beyond the 20 points for each correct answer: a quiz-out without error (achieved through individual accuracy) and a 3rd/4th/5th person bonus (achieved through team participation). This creates a third way to achieve that (through a "streak"), which adds some strategic elements to the scoring because it now matters _when_ you get a question correct. We reward a 3PB because statistically it is hard to do. The same with the proposed 3CB.
2. As a result of this increased opportunity for team points, it gives teams that are trailing late in a quiz another avenue to come back. For example, if a team is down by 70 points on Question 18, a 3rd (or 4th/5th) Person Bonus is their only option. But with a 3rd Consecutive Bonus, 2 quizzers quizzers or even 1 quizzer can make up those 70 points. Teams that have 3 quizzers who can consistently answer questions correctly in a quiz are far less common than those with 2 quizzers who can consistently answer questions correctly in a quiz. Thus, by lowering the participation threshold for bonus points while not lowering the statistical difficulty to earn them, we give teams another route to win.
3. A 3CB gives top quizzers, who can generally choose their questions, the chance to maximize slightly more the points they can contribute to their team by timing those jumps to be on 3 consecutive questions. Of course, timing those jumps in order to achieve a 3CB is a risk, and therefore one which, if undertaken successfully, should be rewarded.
4. Finally, a 3CB would be unique in that its bonus potential can be had _in addition to_other bonuses (e.g. a quiz-out without error and a 3PB). That means a given question, depending on who jumps and when, could be worth 40 points, putting many more quizzes within reach.

I would add to the above advantages two addendums:

1. On the same logic as a 3CB, there should be a 4CB as well worth 20 additional points, it being worth more because it is more difficult. Nothing beyond that 4CB, however, should be offered because a 4CB is the maximum that can be answered correctly by a single quizzer.
2. Like a 3PB, a 3CB or 4CB would count only toward the team score _unless it is the same quizzer who has answered all 3 or 4 consecutive questions_. In such cases, those points would count toward personal points just as the 10 points for a quiz-out without error. Therefore, if a quizzer answered 4 consecutive questions correctly, the most points a quizzer could achieve (theoretically) in a quiz would be 120 points: 80 for four correct answers, +10 for a quiz-out without error, +10 for the 3CB, +20 for the 4CB.

The proposed additions to the rulebook would be as follows for individuals:

> **6.1.1.1. Points Earned**
> +10 points for correct answers on three consecutive questions
> +20 points for correct answers on four consecutive questions

And for teams:

> **6.1.2.1. Points Earned**
> +10 points for correct answers on three consecutive questions by any combination of two or three quizzers.
> +20 points for correct answers on four consecutive questions by any combination of two, three, or four quizzers.

**6.1.2** already says that, 

> All points earned or deducted by an individual are to be counted towards the team's points.

so the additions in 6.1.1.1 already cover the scoring there.

**Comment by scrobiculatus** [comment]
I think I would support this if I had the power to; it is difficult to get 3 or 4 questions in a row and should be rewarded. I have pulled off the four in a row a few times at a district level, which was difficult (even at the district level), and a few more points would not be inappropriate as a reward for doing so. However, I do not think it is fair for the 4CB to be worth 20 points, and the 4PB to be worth only 10. It is also very difficult to get a 4PB in a quiz, and if the 4CB ends up being worth 20 points, then so should the 4PB (in my opinion). If the 4PB is not changed, I would rather have the 4CB also remain at 10 points.)
I do also think that the addition of this rule would make Quizzing more luck-based, with 30 (or 20) points solely depending on when you get your questions. A team that would have gotten 150 points with the current rules (quizzer 1 getting 20, quizzer 2 getting 60, quizzer 3 getting 20) can now get 180 points if four of the six correctly answered questions were consecutive.

**Comment by josiah-leinbach** [comment]
I concur with your observation about the 4PB, which is why I have also proposed that it be changed:

https://github.com/gryphonshafer/Quizzing-Rule-Book/issues/200

I do not think this would make accruing points chance-based for the reason that you outlined: getting 3 or 4 questions correct in a row is very difficult. It is difficult not because it is statistically unlikely (though it is) but because given the question distribution and human psychology, you pull something like that off by having great discipline. In order to get a 3CB, all of the following would need to happen:

1. You get the two previous questions correct. If you miss either one, there is no streak to build upon.
2. Neither of the other two teams gets the jump on the third question. If one of them jumps, even if they get it wrong, the streak is over.
3. You jump on the third question. If you do not jump, the streak is over.
4. You get the question correct.

Now, let's take a look at how all these demonstrate preparation and disciple. First, by taking three-straight questions, you are not simply choosing your jumps based on the question type. That shows individual/team versatility. Second, you have to get at least two of those questions correct while all three teams are in, so you are not simply taking advantage of toss-ups. Third, you show discipline and do not let the hype and emotion cause you to jump too quickly on the third question.

Correct answers in quizzing are not independent events like flipping a coin, but the circumstances and decision-making for every question are influenced by the prior events and the human reaction to those prior events. Thus, a team achieves a 3CB and 4CB not because of chance but because of discipline and skill; and if chance is involved, it is no more chance than in any other quizzing scenario.

**Comment by 24RMiller** [comment]
I have mixed feelings about this rule change and could be persuaded either way. However, I'd say I am a fan of the rule with a slight change. I think the 3CB bonus should only count for team stats, and not individual stats, especially at Internationals. It wouldn't be fair for a quizzer on a stacked team to receive less points than a quizzer on a weak team, merely because the quizzer on the weak team could jump on any specialty, but the quizzer on the strong team is limited, and has to allow their teammates to "assist" them in obtaining the 3CB. I am a big fan of adding more strategy into quizzing, and I think this rule does add more to the team concept of quizzing, but it also provides too much of a benefit for individual quizzers trying to obtain as many individual points as possible. 
However, I could see many situations, such as at the district level, when individual stats are more important, and a quizzer should be rewarded for getting 3 or 4 correct in a row. I have one much more radical idea that I will at least write in here, although it might get shot down right away. What if we added a section in the rulebook called "Optional Bonuses?" In this section multiple scoring ideas such as the one stated above would be included. Other more unique ideas could be included under this section such as an idea we tried in our district that involved an extra 10 points for the first person to quiz out without error in a quiz. Below the section title would say something to effect of: "Meet directors/district coordinators have the option to include any of the following bonuses during a quiz meet for individual scores, team scores, or both. The bonuses used for Internationals will be chosen by the CQLT at the conclusion of the previous year's Internationals (requires 4/6 [maybe all 6] CQLT members to agree in order to add a new bonus)."

This would allow different districts/tournaments to include their own fun twists and pave the way for them to be used at Internationals in the future. My guess is that at first people will not want to include too many of these bonuses at nats, but as time progresses, they could become used at nats. This allows more future flexibility with the rules, while also not making any radical changes immediately. I do realize that districts already have the authority to make their own rules, but I think most of them would be hesitant to add a rule like this without it at least being in the rule book in some form.

**Comment by levikoral** [comment]
I could see how this could work. It would add an additional "benefit" to quizzing, especially, as you said, @josiah-leinbach, if they are losing by a sizable margin. I personally don't have any problem with the idea.

**Comment by jttower** [comment]
Overall I like this idea.  I do hear what @24RMiller is saying, so we should talk about the pros/cons of applying the bonus to the personal vs team score.  Right now I am leaning towards @josiah-leinbach  with adding points to both personal and team scores, I just want to make sure we've thought through possible scenarios at IBQ and whether this would be an unfair advantage somehow.

**Comment by josiah-leinbach** [comment]
@24RMiller I understand the arguments against counting such points toward one’s personal score, but here are the reasons I am unpersuaded.

Doing a Fermi estimate here, I am going to guess that, on the high end, there were around 25 quiz-outs recorded at Internationals last year and around 35 three-correct scores. Take then the approximately 75 quizzers and multiply them by their 12 prelim quizzes and we get 900 individual score counts. So with a 3CB/4CB counting toward personal scores, we are talking about an event whose theoretical ceiling at Internationals is applicable around 7% of the time (with a 4CB have a 3% ceiling). Next, ask how many of those 60 instances (or 7%) were instances where at least three of the correct questions were answered on consecutive questions. I do not have specific numbers here, but I am going to guess that the occurrence was very, very low; but let us assume it happened 6 times (which still seems too high). Thus, assuming the highest number of occurrences possible, the 3CB/4CB applying to an individual score would have been 0.67% of all scores at Internationals, with a 4CB happening maybe once (0.01%) if at all.

Now it might be argued that if you create the incentive to get points for correct questions, you would increase the number of occurrences from what has happened apart from that incentive. True—but how much higher would such an incentive realistically raise that ceiling? 1% perhaps? If it is already so difficult to get 3 or 4 correct answers, an incentive structure like this one could hardly alter the distribution too much. Therefore, given the rarity of these events, rewarding them is both fitting while at the same time will not alter the dynamics of individual scoring too much.

As to the argument that it would favor strong quizzers on weaker teams, we already have these sorts of instances occur under the current system. Suppose one my team I have Quizzer A (a FTV quizzer) with 0-1 and Quizzer B (a non-FTV quizzer) with 3-0. Now suppose Question 16 is a FTV, I have a free error, and Quizzer B has more than a decent shot at getting a FTV correct. I am far more inclined to have Quizzer B take that jump than Quizzer A, given the risk/downside (0 vs. -10) and reward/upside (20 vs. 30). Did that coaching decision cost Quizzer A a potential FTV? Perhaps, but most people would recognize that as a reasonable, strategic calculation, even if they disagreed with it. Assuming someone has a quizzer who can, at will, get 3 or 4 correct in a row and earn 120 points, that not only becomes part of my strategy but also by definition moves that team from a “weak team” to a “strong team” (in terms of expected points).

**Comment by josiah-leinbach** [comment]
@kclimenhaga The key factor, however, is that you do _know_ whether a team is going to get 30-40 points on a question, only that they have the _potential_ to do so. We already have these scenarios with 3rd/4th/5th person bonuses, quiz-outs without error, the “up by 30 on Question 20A” situation, and to a less magnitude (but at greater frequency) with stealing normal questions. In fact, one could argue that such a 3CB/4CB actually cuts the other way, namely that it incentivizes the team trying to earn the consecutive bonus to jump more quickly. I think is the far more likely of scenario, but in similar circumstances we would simply describe that as a calculated risk, not unlike me telling my quizzer with 3-0 wrong to be more aggressive on a question and get the 20+10 points for it, or if we need 50 points on Questions 19 and 20. Thus, if a team thinks that it is to its advantage to jump more aggressively on a question than it otherwise normally would, then so be it. And given that such scenarios already exist in the nature of quizzing, there is no real reason to avoid introducing another. 

Simply put, there is no incentive to “take an error” as such, but only to “take a jump which could result in an error”. That distinction in crucial, because in the former you _want_ to error, whereas in the latter you simply want to nullify the ability of the other team’s odds of getting the question correct (however large or small those odds are).

**Comment by kclimenhaga** [comment]
The difference with team bonuses and quiz-outs is that in order to block a team from getting those, you would need to burn every jump, which of course isn't a good strategy. If a quizzer has gotten 3 correct by question 6, then you can't block the quiz out without error by burning question 7.

Say Team A has gotten questions 1-3 correct, and question 4 is an INT. Strategy for Team A is to jump aggressively, but not so aggressively that they will get it wrong (so, say, around 1.5 syllables). Strategy for Teams B and C is to jump on formation and get the error, because otherwise they risk letting Team A get 40 points on that question. If it's a REF, strategy is to jump on the chapter number and pick a random verse to quote, with a very low likelihood of getting a correct answer. Of course, they don't "want" to error, but they want to jump at a speed that will almost certainly result in an error. To me, that's different than just jumping aggressively, or "taking a jump which _could_ result in an error."

> And given that such scenarios already exist in the nature of quizzing, there is no real reason to avoid introducing another.

I disagree. If it's a bad thing to have situations where blocking another team is the best strategy, then we should keep those situations to a minimum.

**Comment by JoshJetto** [comment]
I am in agreement with @kclimenhaga on being against this rule change for the reasons she gives. I am also opposed to this rule change for at least three other reasons: 1) it is an unnecessary over-complication of the scoring system, 2) it over-advantages a team with a strong individual quizzer and thus, makes the ability of the rest of the team less necessary for the team to win, and 3) I don't think we need to add dramatic ways to catch up at the end of the quiz nor (and this is the more likely scenario) ways for a team that would be having a very good quiz under our current scoring rules to basically become uncatchable with additional scoring bonuses.

**Comment by LukeBraisted** [comment]
With some modifications to clarify the rule and minimize the chances of burning questions, consecutive bonuses for both teams and individuals would positively affect district, zone, and invitational quizzing.

**Benefits of the rule**

Consecutive bonuses would improve individual scoring in certain ways. The top quizzers’ rankings in lower-intensity competitions would be based more on speed and knowledge of different question types, rather than slight variations in the number of errors made. The rule would also help prevent ties among the top quizzers, such as the four-way tie for first place at the most recent Pittsburgh invitational.

The proposed rule would add to the competition in quizzes. I think the opportunities for more teams to potentially win would make the quizzes a little more exciting, even though consecutive bonuses could also sometimes put strong teams out of reach. Adding consecutive bonuses would increase the speed and intensity of quizzes. The new rule would provide a challenge for higher-level quizzers in district competitions and encourage them to know the material even better.

Another benefit of consecutive bonuses is that they encourage the ability to do more question types. For example, the new rule offers both team and individual advantages if a quizzer who does well at INTs and MAs learns to do memory questions as well. This increased knowledge of the material is something worth rewarding.

**Objections to the rule**

A potential concern about the proposed rule is whether it could negatively affect the International competition. Looking through the videos from five tournament quizzes at Internationals, I found just two times when a team got the jumps on three consecutive questions. Only one of the two times would have resulted in a 3CB if the proposed rule was in effect. (CMD1 had three consecutive correct answers from three different quizzers in quiz D.) So consecutive bonuses would probably be rare enough that they would not be a big change for Internationals.

One issue that has been mentioned is the potential for deliberate errors to break another team’s streak. These could be mostly prevented with an addition to the rule. If one team gets a 3CB and another team errs on the next question, the team that got the 3CB could be given a 10-point bonus if it gets the next question that has all teams jumping. (The same bonus would apply to an individual’s score if the individual gets the 3CB and the other question.) Then there would still be an advantage to a 4CB over an error from another team, but the other teams would not benefit by getting an error. Burning a question would be an error (usually -10) to reduce the value of a _possible_ bonus for one of the other teams by 10 points. If the jumping is fast enough that blocking a streak before the 3CB requires burning a question, the chance is fairly low that the team trying for a streak would actually get the next two questions correct (assuming the remaining quizzers on the two teams have a comparable level of knowledge). If 4PBs remain as 10-point bonuses, it would be more consistent and simpler to have 4CBs also be 10-point bonuses. In that case, the addition described above would not be needed.

Some would say that the proposed rule gives too much of an advantage to a team with just one strong quizzer. Consecutive bonuses give an advantage to teams that are strong on many question types. Although this strength could often come from one quizzer, it could also come from multiple quizzers. Even with a 4CB, a team with only one quizzer answering is limited to 140 points (except for bonus questions). It is extremely unlikely that any individual would always get a perfect 4CB, and, based on team stats from invitationals and various districts, the top teams almost always average more than 140 points per quiz. So one quizzer generally could not score enough to single-handedly achieve a first-place team; his teammates would need to contribute. In a tournament-based meet, 140 points per quiz might be enough to win, but achieving 4CBs with only one quizzer would be even more unlikely.

A disadvantage to consecutive bonuses is that they add more complexity to the scoring system, a part of quizzing that many outsiders and even newer quizzers do not fully understand. A thorough understanding of scoring, however, is not necessary to watch quizzing or even to participate. While the proposed rule could cause some difficulties, it would not be terribly hard for most of the coaches and experienced quizzers—the people who would be most affected by the rule and would need to understand it. If the rule brings significant benefits, I think these are worth the added complexity.

One argument against the consecutive bonus rule is that it disadvantages quizzers on strong teams. In lower-intensity competitions where streaks are more likely, stacked teams are less common. If two strong quizzers do end up on the same team in one of these competitions, they could agree not to break each other’s individual streaks. This would minimize any disadvantage resulting from the new rule.

The consecutive bonus rule does have the downside that it could cause more differences between what would benefit team scores and individual scores. Getting an individual streak requires jumping on whatever question types come up, but it might be better for the team to have a different quizzer jump on some of those types. This is unlikely to be an issue at the highest levels of quizzing with a lot of specialization since individual consecutive bonuses would be extremely rare. But at lower levels, an individual who is strong on all types could potentially have a teammate who is strong on only one or two types. If one of those types comes up, the individual’s streak might need to be interrupted to allow his teammate to get a question. This could cause some imbalances in the individual scores, but it would probably not be an extremely common situation. Counting consecutive bonuses only for the team score would not solve this issue. There would still be conflicting team and individual incentives, since trying to get a consecutive bonus for the team increases the risk of errors for the individuals.

**Suggested clarifications**

The proposed rule could use some additional clarifications. First, a definition of consecutive questions should be added to specify that bonus questions do not count toward streaks and that alphanumeric questions are not consecutive to the previous question number. (18A is not consecutive to 17 or 17A.) Second, the rules should indicate that if one question gives both an individual 3CB and a team 4CB, the team receives 20 bonus points, not 30. A 4CB in which one quizzer gets the second, third, and fourth questions should not give a higher team reward than other 4CBs.

**Conclusion**

In conclusion, the downsides of consecutive bonuses are not too severe, and the issue of question-burning can be prevented. The district-level positive impacts of improving the competition and encouraging better material knowledge make the new rule worthwhile.

**Comment by ARMediting** [comment]
I haven't considered this in any depth other than reading this thread, but my first thought is to agree with Kristen and Josh, that it provides a very obvious scenario where burning a question (with little regard for needing to get it correct) is the unarguably best move. When a team gets several questions correct in a row, the momentum is already with them, and the other teams are already trying to disrupt them. 

This is a more subjective reason not to have it, but when one team gets several questions in a row early in a quiz, that puts a lot of pressure on the other teams and can make a quiz feel quickly out of reach. Extra bonuses would speed up or increase the chances that a team could rack up an insurmountable lead, which is discouraging. Of course this rule could also aid a team making a late comeback, and it's debatable which of these two scenarios happens more. But having certain questions worth up to 60 or 70 points if various bonuses stack feels too extreme, and much more complex to coach for.

**Comment by josiah-leinbach** [comment]
For the record, this tested poorly in a our beta-test tournament with it in October 2023, so I would need to overhaul how this works.
